### DISCUSSION TITLE
React 18

### DISCUSSION THREAD

**Discussion by theohagos** [chakra-ui/chakra-ui#5794] [Q&A]
Chakra  Drawer seems to have an issue with React 18, it seems to block my app even after dismissing it. Has anyone come across this?

**Comment by tombohub** [comment]
react 18.1.0, chakra  2.1.0, nothing unusual here

**Reply by theohagos** [reply]
what does that even mean?

**Reply by tombohub** [reply]
It means I have used those 2 and didn't notice anything wrong. Drawer working as expected.

**Reply by theohagos** [reply]
ah, thanks for the quick reply. I haven't changed anything just upgraded to React 18 and Chakra 2.0.0, maybe will try Chakra 2.1.0 and see if that helps.

**Reply by tombohub** [reply]
I think I was having problems with Chakra 2.1.0 and React 17. Pretty much you have to use React 18 with 2.1.0
